CopilotKit/packages/runtime/src/lib/runtime/remote-lg-action.ts

```
@@ -888,3 +791,170 @@ function getSchemaKeys(graphSchema: GraphSchema): SchemaKeys {
+    console.warn(
+      `Cannot use the threadId ${threadId} with LangGraph Platform. Must be a valid UUID.`,
+    );
+  }
+
+  return threadId;
+}
+
+async function getOrCreateThreadAndReturnState(
+  client: LangGraphClient,
+  threadId: string,
+): Promise<ThreadState<{}>> {
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)